packages/unison/unison.2.52.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "File-synchronization tool for Unix and Windows"
+description: """\
+Unison is a file-synchronization tool for Unix and Windows. It allows
+two replicas of a collection of files and directories to be stored on
+different hosts (or different disks on the same host), modified
+separately, and then brought up to date by propagating the changes in
+each replica to the other."""
+maintainer: "juergen@hoetzel.info"
+authors: ["Trevor Jim" "Benjamin C. Pierce" "Jérôme Vouillon"]
+license: "GPL-3.0-or-later"
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "https://github.com/bcpierce00/unison/issues"
+depends: [
+  "ocaml" {>= "4.03"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
```
As suggested in the pr, is needed to fix
```
#=== ERROR while compiling unison.2.52.0 ======================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.05.0 | pinned(https://github.com/bcpierce00/unison/archive/refs/tags/v2.52.0.tar.gz)
# path                 ~/.opam/4.05/.opam-switch/build/unison.2.52.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p unison -j 31
# exit-code            1
# env-file             ~/.opam/log/unison-21055-682ec9.env
# output-file          ~/.opam/log/unison-21055-682ec9.out
### output ###
# (cd _build/default && /home/opam/.opam/4.05/bin/ocamlopt.opt -w -40 -w -3-6-9-10-26-27-32-34-35-38-39-50-52 -warn-error -3-6-9-10-26-27-32-34-35-39-50-52 -no-strict-sequence -g -I src/.unison_lib.objs/byte -I src/.unison_lib.objs/native -I src/lwt/.lwt_lib.objs/byte -I src/lwt/.lwt_lib.objs/native -intf-suffix .ml -no-alias-deps -o src/.unison_lib.objs/native/umarshal.cmx -c -impl src/ubase/umarshal.ml)
# File "src/ubase/umarshal.ml", line 34, characters 12-30:
# Error: Unbound value Bytes.get_int64_be
```

Not committing, since there is some ongoing discussion on this PR that may require further changes